### PR TITLE
Cherry-pick #22821 to 7.x: Apache status: adjust field collection in fleet mode

### PR DESCRIPTION
**Pull request by mtojek**
Cherry-pick of PR #22821 to 7.x branch. Original message: 

## What does this PR do?

This PR modified field collection log in the Apache module, "status" metricset, so they're collected according to expectations of the Apache Integration.

`apache.status.hostname` -> `host.hostname` (already present)
~`apache.status.uptime.uptime` - >`host.uptime`~ (decided not to proceed with this change. `host.uptime` has a different meaning)

## Why is it important?

Migration to Beats (https://github.com/elastic/integrations/issues/370)

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] ~I have made corresponding changes to the documentation~
- [ ] ~I have made corresponding change to the default configuration files~
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

(there is an integration test)

## Related issues

- https://github.com/elastic/integrations/issues/370

### CODE REVIEW

**Comment by mtojek** [comment]
jenkins run the tests please

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -614,6 +614,18 @@ same journal. {pull}18467[18467]
...
+*Packetbeat*
+
+- Add an example to packetbeat.yml of using the `forwarded` tag to disable
+  `host` metadata fields when processing network data from network tap or mirror
+  port. {pull}19209[19209]
+- Add ECS fields for x509 certs, event categorization, and related IP info. {pull}19167[19167]
+- Add 100-continue support {issue}15830[15830] {pull}19349[19349]
+- Add initial SIP protocol support {pull}21221[21221]
+- Add support for overriding the published index on a per-protocol/flow basis. {pull}22134[22134]
+- Change build process for x-pack distribution {pull}21979[21979]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc:628]: There are extra lines here.

**Review by jsoriano — CHANGES_REQUESTED** [review]

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (mtojek) [reply CHANGELOG.next.asciidoc]: Missed these. Fixed!

**Review by jsoriano — APPROVED** [review]
:+1:

**Pull request merged**